global.json

```diff
@@ -1,6 +1,7 @@
-{
-  "sdk": {
-    "version": "3.1.100",
-    "rollForward": "latestFeature"
-  }
-}
\ No newline at end of file
+
+{ 
+  "sdk": {	
+    "version": "3.1.400",	
+    "rollForward": "latestFeature"	
```

Comment:
As, net5.0 is stable now, maybe we could switch the rollForward to `latestMajor`